aodndata/moorings/products_handler.py

```diff
@@ -246,14 +246,19 @@ def _add_to_collection(self, product_url):
         product_file.publish_type = PipelineFilePublishType.HARVEST_UPLOAD
         self.file_collection.add(product_file)
 
+    def _input_list_for_variables(self, *variables):
+        """Return a list of input files containing any of the given variables"""
+        input_list = [f for f, f_vars in self.input_file_variables.items()
+                      if any(v in f_vars for v in variables)
```

Comment:
would it be `all` instead of ```any``` to avoid the 0.0001% chance of a file without UCUR/VCUR but with VCUR/UCUR to be included and raise a runtime error down the line?

Reply:
@mhidas - just confirm the above and free to merge

Reply:
Ideally, yes, all velocity files _should_ have UCUR _and_ VCUR (and also WCUR), and in fact pretty much all of them do. However, if there happen to be any files with only one and not the other, we want to know about it, not just silently ignore it by not selecting it here.
Also, the product generating code [checks](https://github.com/aodn/python-aodntools/blob/ae8b06eff10a2143f0f7766a82844c433af4634c/aodntools/timeseries_products/velocity_aggregated_timeseries.py#L20) each input file before processing and excludes files that don't meet the minimum requirements, so these won't lead to runtime errors. Instead they are reported on output and also listed in a global attribute of the product file.